recipes/orocos_kdl/bldpkg.bat

```diff
@@ -0,0 +1,23 @@
+:: MSVC is preferred.
+set CC=cl.exe
+set CXX=cl.exe
```

Comment:
This doesn't seem correct

Reply:
I think the feedback came in another PR too. Here is the comments: https://github.com/conda-forge/staged-recipes/pull/10985#discussion_r387797675

Just wondering should I fall back to use "NMake Makefiles" generator? (It is fine, however which doesn't make use of the multi-core, so it generally slows the build.) Or any other guidance ?

Reply:
CC @conda-forge/core